art/psa-attester.ascii-art

```diff
@@ -24,3 +24,6 @@
 | | Target Environment                                            | |
 | '---------------------------------------------------------------' |
 '-------------------------------------------------------------------'
+Legenda:
```

Comment:
Is _**legenda**_ correct? Not legend? I'm not familiar with what RFCs use.

Reply:
addressed in [2548f51](https://github.com/thomas-fossati/draft-psa-token/pull/125/commits/2548f51f98e1960ffea60992c14a68140e3281a2)